### ISSUE TITLE
How to disable ipv6 ?

### ISSUE THREAD

**Issue by Fridus** [pritunl/pritunl#314]
CentOS 7 docker, Installed from source
```
[summer-plains-5213][2016-04-08 15:51:18,020][WARNING] Failed to load pymongo c bindings
[summer-plains-5213][2016-04-08 15:51:18,021][WARNING] Failed to load bson c bindings
[summer-plains-5213][2016-04-08 15:51:18,023][INFO] Starting debug server
[summer-plains-5213][2016-04-08 15:51:20,211][ERROR] Popen returned error exit code
  cmd         = ['ip6tables', '-I', 'INPUT', '-i', 'tun0', '-j', 'ACCEPT', '-m', 'comment', '--comment', 'pritunl_5707b83d545bc2000a460dc6']
  return_code = 3
Process stderr:
  ip6tables v1.4.21: can't initialize ip6tables table `filter': Table does not exist (do you need to insmod?)
  Perhaps ip6tables or your kernel needs to be upgraded.
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 784, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/server/instance.py", line 736, in _run_thread
    self.iptables.upsert_rules()
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/iptables.py", line 519, in upsert_rules
    self._insert_iptables_rule(rule, ipv6=True)
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/iptables.py", line 463, in _insert_iptables_rule
    ['ip6tables' if ipv6 else 'iptables', '-I'] + rule)
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/utils/misc.py", line 146, in check_output_logged
    stderr=stderrdata,
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/logger/__init__.py", line 55, in error
    kwargs['traceback'] = traceback.format_stack()
[summer-plains-5213][2016-04-08 15:51:20,213][ERROR] Failed to insert iptables rule, retrying...
  rule = ['INPUT', '-i', 'tun0', '-j', 'ACCEPT', '-m', 'comment', '--comment', 'pritunl_5707b83d545bc2000a460dc6']
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 784, in __bootstrap
    self.__bootstrap_inner()
  File "/usr/lib64/python2.7/threading.py", line 811, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 764, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/server/instance.py", line 736, in _run_thread
    self.iptables.upsert_rules()
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/iptables.py", line 519, in upsert_rules
    self._insert_iptables_rule(rule, ipv6=True)
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/iptables.py", line 471, in _insert_iptables_rule
    rule=rule,
  File "/usr/lib/python2.7/site-packages/pritunl-1.21.939.33-py2.7.egg/pritunl/logger/__init__.py", line 55, in error
    kwargs['traceback'] = traceback.format_stack()
```

**Comment by zachhuff386** [comment]
IPv6 can be disabled in the server settings. The error above is fixed in v1.21.941.39

**Comment by Fridus** [comment]
Thx,
I can disable ipv6 with file `pritunl.conf` ?
The documentation about settings is blur for me
